zone/raids.cpp

```diff
@@ -1176,8 +1271,13 @@ void Raid::SendMakeLeaderPacket(const char *who) //30
 
 void Raid::SendMakeLeaderPacketTo(const char *who, Client *to)
 {
-	if(!to)
+	if (!to) {
 		return;
+	}
+
+	if (RuleB(Bots, Enabled) && members[GetPlayerIndex(who)].IsBot) {
```

Comment:
This seems crash prone, we should check that player index is valid and not out of bounds